soccer/src/soccer/WorldState.hpp

```diff
@@ -3,6 +3,9 @@
 #include <Geometry2d/Pose.hpp>
 #include <rj_common/time.hpp>
 #include <rj_constants/constants.hpp>
+#include <rj_msgs/msg/detail/ball_state__builder.hpp>
+#include <rj_msgs/msg/detail/robot_state__builder.hpp>
+#include <rj_msgs/msg/detail/world_state__builder.hpp>
```

Comment:
Include regular message headers instead of __builders.

Reply:
done.